gym_solo/envs/solo8v2vanilla.py

```diff
@@ -23,6 +23,20 @@
 @dataclass
 class Solo8VanillaConfig(Solo8BaseConfig):
   urdf_path: str = 'assets/solo8v2/solo.urdf'
+  starting_joint_pos = {
```

Comment:
How did you come up with these values?

Reply:
Experimentally found. Keeping it like this cause it's a "config" and it unique to the specific model. The only way to get around this would be to modify the file in solidworks as the urdf can't easily be edited to reorient all of the joints.

Reply:
Got it